draft/spec/index.html

```diff
@@ -1052,8 +1052,9 @@ <h2>Root Structure</h2>
         <h2>Root Conformance Declaration</h2>
         <p>
           The OCFL version declaration <span id="E075">MUST</span> be formatted according to the [[!NAMASTE]]
-          specification. It <span id="E076">MUST</span> be a file in the base directory of the <a>OCFL Storage Root</a>
-          giving the OCFL version in the filename. The filename <span id="E077">MUST</span> conform to the pattern
+          specification. There <span id="E076">MUST</span> be exaclty one version declaration file in the base
```

Comment:
typo: `exaclty`

Reply:
Thanks, fixed in [5992508](https://github.com/OCFL/spec/pull/585/commits/5992508e8b81f3942d2374de253a9cfe7353c097)